lib/src/view.rs

```diff
@@ -46,7 +45,7 @@ impl View {
         }
     }
 
-    pub fn wc_commit_ids(&self) -> &HashMap<WorkspaceId, CommitId> {
+    pub fn wc_commit_ids(&self) -> &BTreeMap<WorkspaceId, CommitId> {
```

Comment:
nit: we can remove `.sorted()` from callers.

Reply:
Yes, good idea.

There's more complicated code in `repo.rs` around line 1666 that might be possible to simplify, I can add a TODO there as well.